biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -530,7 +530,9 @@ subroutine FatesPlantRespPhotosynthDrive (nsites, sites,bc_in,bc_out,dtime)
                                                         bc_in(s)%forc_pbot,                 &  ! in
                                                         bc_in(s)%cair_pa(ifp),              &  ! in
                                                         bc_in(s)%oair_pa(ifp),              &  ! in
+                                                        bc_in(s)%rb_pa(ifp),                &  ! in                                                        
                                                         btran_eff,                          &  ! in
+                                                        psi_leaf,                           &  ! in                                                        
```

Comment:
I believe we call this routine for both hydro and non-hydro versions of the code.  One solution is to initialize psi_leaf with a flag value for non-hydro cases, so we can option to turn off its use when we enter LeafLayerPhotosynthesis()